torch/testing/_internal/hypothesis_utils.py

```diff
@@ -346,22 +346,33 @@ def tensor_conv(
 
     return X, W, b, groups, tr
 
+
 # We set the deadline in the currently loaded profile.
 # Creating (and loading) a separate profile overrides any settings the user
 # already specified.
 hypothesis_version = hypothesis.version.__version_info__
```

Comment:
Nit should probably use importlib version utils now

Reply:
Thank you, have updated it.